include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -15,2970 +15,7 @@
-         (typename std::decay_t<decltype(*this)>::ExcMatrixDoesNotMatch()));
-
-  Assert(!this->has_children(), ExcMessage("Cell must be active."));
-
-  const unsigned int n_dofs = local_source.m();
-
-  internal::DoFAccessorImplementation::Implementation::dof_index_vector_type
-    dof_indices(n_dofs);
-  internal::DoFAccessorImplementation::get_cell_dof_indices(
-    *this, dof_indices, this->active_fe_index());
-
-  // distribute cell matrix
-  for (unsigned int i = 0; i < n_dofs; ++i)
-    global_destination.add(dof_indices[i],
-                           n_dofs,
-                           dof_indices.data(),
-                           &local_source(i, 0));
-}
-
-
-
-template <int dimension_, int space_dimension_, bool level_dof_access>
-template <typename number, typename OutputMatrix, typename OutputVector>
-inline void
-DoFCellAccessor<dimension_, space_dimension_, level_dof_access>::
-  distribute_local_to_global(const FullMatrix<number> &local_matrix,
-                             const Vector<number>     &local_vector,
-                             OutputMatrix             &global_matrix,
-                             OutputVector             &global_vector) const
-{
-  Assert(this->dof_handler != nullptr,
-         (typename std::decay_t<decltype(*this)>::ExcInvalidObject()));
-  Assert(local_matrix.m() == this->get_fe().n_dofs_per_cell(),
-         (typename std::decay_t<decltype(*this)>::ExcMatrixDoesNotMatch()));
-  Assert(local_matrix.n() == this->get_fe().n_dofs_per_cell(),
-         (typename std::decay_t<decltype(*this)>::ExcVectorDoesNotMatch()));
-  Assert(this->dof_handler->n_dofs() == global_matrix.m(),
-         (typename std::decay_t<decltype(*this)>::ExcMatrixDoesNotMatch()));
-  Assert(this->dof_handler->n_dofs() == global_matrix.n(),
-         (typename std::decay_t<decltype(*this)>::ExcMatrixDoesNotMatch()));
-  Assert(local_vector.size() == this->get_fe().n_dofs_per_cell(),
-         (typename std::decay_t<decltype(*this)>::ExcVectorDoesNotMatch()));
-  Assert(this->dof_handler->n_dofs() == global_vector.size(),
-         (typename std::decay_t<decltype(*this)>::ExcVectorDoesNotMatch()));
-
-  Assert(!this->has_children(), ExcMessage("Cell must be active."));
-
-  const unsigned int n_dofs = this->get_fe().n_dofs_per_cell();
-  internal::DoFAccessorImplementation::Implementation::dof_index_vector_type
-    dof_indices(n_dofs);
-  internal::DoFAccessorImplementation::get_cell_dof_indices(
-    *this, dof_indices, this->active_fe_index());
-
-  // distribute cell matrices
-  for (unsigned int i = 0; i < n_dofs; ++i)
-    {
-      global_matrix.add(dof_indices[i],
-                        n_dofs,
-                        dof_indices.data(),
-                        &local_matrix(i, 0));
-      global_vector(dof_indices[i]) += local_vector(i);
-    }
-}
-
-
-
-DEAL_II_NAMESPACE_CLOSE
+#warning \
+  "The use of this header file is deprecated. Just include <deal.II/dofs/dof_accessor.h>."
```

Comment:
We do
```suggestion
#include <deal.II/base/config.h>

DEAL_II_WARNING("The use of this header file is deprecated. Just include <deal.II/dofs/dof_accessor.h>.")
```
in similar places.